proteuslib/unit_models/reverse_osmosis_0D.py

```diff
@@ -891,7 +890,14 @@ def eq_over_pressure_ratio(b, t):
                     * (b.feed_side.properties_out[t].pressure_osm
                     - b.permeate_side.properties_out[t].pressure_osm))
 
-    def initialize(blk, initialize_guess=None, state_args=None, outlvl=idaeslog.NOTSET, solver=None, optarg=None):
+    def initialize(blk,
+                   initialize_guess=None,
+                   state_args=None,
+                   outlvl=idaeslog.INFO,
```

Comment:
We generally use NOTSET here, as that tell IDAES to use whatever default the user has specified in their global configuration.